pkgs/os-specific/linux/hp-vendor/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, kernel }:
+stdenv.mkDerivation rec {
+  name = "hp-vendor-${version}-${kernel.version}";
+  version = "master_jammy";
```

```suggestion
  pname = "hp-vendor";
  version = "unstable-REPLACE";
```

Would be good if the pname and attr would include pop `pop-hp-vendor`